### PR TITLE
docs(feg): Tier is needed to setup an feg

### PR DESCRIPTION
**Pull request by crasu**
Signed-off-by: Christian Kraemer <Christian.Kraemer@tngtech.com>

## Summary

If you have not installed the agw before. No tier is set up in the feg db.

## Test Plan

Manual test

### CODE REVIEW

**Review by Neudrino — APPROVED** [review]

File: docs/readmes/feg/deploy_install.md [hunk 1]
```diff
@@ -143,9 +143,10 @@ To register the FeG, go to the Orchestrator's Swagger UI in your browser.
 (i.e. <https://controller.url.sample:9443/swagger/v1/ui/>).
 
 Now, create a Federation Network. This is found at `/feg` under the
-**Federation Networks** section. Then register the gateway under the
-**Federation Gateway** section at `/feg/{network_id}/gateways` using the
-network ID of the Federation Network and the hardware ID and challenge key
+**Federation Networks** section. If you have not registered any gateways before you
```
Comment (voisey) [inline docs/readmes/feg/deploy_install.md]:
```suggestion
**Federation Networks** section. If you have not registered any gateways before, you
```

File: docs/readmes/feg/deploy_install.md [hunk 2]
```diff
@@ -143,9 +143,10 @@ To register the FeG, go to the Orchestrator's Swagger UI in your browser.
...
+must setup a tier under the **Orchestrator** section at `/networks/{network_id}/tiers`.
```
Comment (voisey) [inline docs/readmes/feg/deploy_install.md]:
```suggestion
must set up a tier under the **Orchestrator** section at `/networks/{network_id}/tiers`.
```

File: docs/readmes/feg/deploy_install.md [hunk 3]
```diff
@@ -143,9 +143,10 @@ To register the FeG, go to the Orchestrator's Swagger UI in your browser.
...
+Then register the gateway under the **Federation Gateway** section at `/feg/{network_id}/gateways`
```
Comment (voisey) [inline docs/readmes/feg/deploy_install.md]:
```suggestion
Then, register the gateway under the **Federation Gateway** section at `/feg/{network_id}/gateways` 
```

**Comment by voisey** [comment]
The markdown linter isn't happy so please run the `make precommit_fix` command described here: https://github.com/magma/magma/wiki/Contributing-Documentation#precommit

**Pull request merged**